api/config/swagger.py

```diff
@@ -378,7 +378,7 @@ def schema_view_swagger(request):
             'Questions': {
                 'Post question': coreapi.Link(
                     # fetch meetup id from the database
-                    url='/api/meetup/{meetup_id}/question/',
+                    url='/api/meetups/{meetup_id}/question/',
```

Comment:
Use plural on the endpoint as discussed on the slack channel.

Reply:
working on this too